File: java-go-dubbo/api-model/src/main/java/com/dubbo/demo/service/DataService.java
package com.dubbo.demo.service;

import com.dubbo.demo.model.RequestData;
import com.dubbo.demo.model.RequestListData;
import com.dubbo.demo.model.ResponseData;
import com.dubbo.demo.model.ResponseListData;

/**
 * 数据接口
 */
public interface DataService {
    public ResponseData DataList(RequestData data);
    public ResponseListData DataListMethod(RequestListData data);
}

File: go-java-dubbo/src/api-model/service/DataService.go
package service

import (
	"context"
	"go-java-dubbo/api-model/model"
)

type DataService interface {
	DataList(ctx context.Context, data *model.RequestData) (*model.ResponseData, error)
	DataListMethod(ctx context.Context, data *model.ResponseListData) (*model.ResponseListData, error)
}
